pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-headers.adoc

```diff
@@ -51,7 +51,7 @@ async fn response_filter(response_state: ResponseState, _config: &Config) {
 }
 ----
 
-Envoy handles the method, scheme, path, authority, and status codes as headers. Modify the headers by accessing the request's `:method`, `:scheme`, `:path`, `:authority`, and `:status`. PDK provides methods for access:
+Envoy handles the method, scheme, path, authority, and status codes as headers. PDK provides the following methods to access and modify the request's `:method`, `:scheme`, `:path`, `:authority`, and `:status` headers:
```

Comment:
some of them are not actually headers, but pseudo headers defined by Envoy. Eg: https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers.

We could call them Envoy psuedo-headers I guess.